registry.py

```diff
@@ -1150,13 +1152,12 @@ def environ_from_url(path):
 
 
 def get_path_info_params(yaml_text):
-    sources = yaml_text['sources']['default_source']
     bbox_req = '-180,-90,180,90'
-    if 'coverage' in sources:
-        coverage = yaml_text['sources']['default_source']['coverage']
-        bbox_req = ','.join([str(f) for f in coverage['bbox']])
 
-    if 'layers'in yaml_text:
+    if 'services' in yaml_text:
+        bbox_req = yaml_text['services']['wms']['bbox']
```

Comment:
yes, it's better to look for the bbox in the output than in the coverage as input